tests/resources/sound_datasets/esc50_download/LICENSE

```diff
@@ -0,0 +1,2114 @@
+# ESC-50: Dataset for Environmental Sound Classification
```

Comment:
I'm not sure why this file is here?

Reply:
Not very sure either. I think it shows up when I run tests. I will remove it before committing.